lib/Backend/Commands.hs

```diff
@@ -87,7 +88,27 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: Member BackendEffect r => EntryPath -> Sem r (Maybe EntryPath)
```

Comment:
Since the `r` type variable in `createCmd` and in `checkForEntriesInEntryPath` are supposed to unify, you could do this:

1. Add a `forall r.` to `createCmd`
2. In `checkForEntriesInEntryPath`, remove the `Member BackendEffect r => ` constraint.

Thanks to [`ScopedTypeVariables`](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/scoped_type_variables.html), GHC will know the `r` in both functions is the same, so any constraints applied to `r` in `createCmd` will also be observed in `checkForEntriesInEntryPath`.

This way, we avoid repeating constraints.